src/services/GithubService.js

```diff
@@ -0,0 +1,16 @@
+import Axios from "axios";
+
+export default {
+    getPullRequests: (username) => {
+        const query = "type:pr author:" + username + " -user:" + username;
+        const url = "https://api.github.com/search/issues";
+
+        return Axios.get(url, {
+            params: { q: query }
+        });
+    },
+    getPullRequestInfo: (issueUrl) => {
+        const url = issueUrl.replace("/issues/", "/pulls/");
```

Comment:
There's a bad practice. 

Check this URL: `https://api.github.com/search/issues?q=type:pr+author:lazyc97+-user:lazyc97` 

Each item already has  pull URL `pull_request.url`

Reply:
Oh, I just copied from @ducthienbui97 's code, fixed it.